File: 1. Two Sum/two_sum.swift
func get_target (nums: Array<Int>, target: Int) -> Array<Int>{
	var len_nums: Int = nums.count;
	var out: Array<Int> = [-1, -1];
	
	for i in 0..<len_nums{
		var exit_loop: Bool = false;
		for j in 0..<len_nums{
			if (i == j){
				continue;
			} else {
				if ((nums[i] + nums[j]) == target){
					out[0] = nums[i];
					out[1] = nums[j];
					exit_loop = true;
				}
			}
		}
		if (exit_loop){
			break;
		}
	}
	
	return (out);
}

var number_list: Array<Int> = [1,2,3,4,5,6,7,8,9];
var target: Int = 16;
var indices: Array<Int> = [-1, -1];

indices = get_target(nums: number_list, target: target);

for i in 0..<indices.count{
	print(indices[i]);
}

File: 1. Two Sum/two_sum.php
<?php
function getTarget ($nums, $target){
	$lenNums = count($nums);
	$out = null;
	
	for ($a = 0; $a < $lenNums; $a++){
		$exitLoop = false;
		for ($b = 0; $b < $lenNums; $b++){
			if ($b == $a){
				continue;
			} else {
				if (($nums[$a] + $nums[$b]) == $target){
					$out = [$a,$b];
					$exitLoop = true;
					break;
				}
			}
		}
		
		if ($exitLoop){
			break;
		}
	}
	
	return ($out);
}

$numberList = [1,2,3,4,5,6,7,8,9];
$target = 16;

$indices = getTarget($numberList, $target);

echo(json_encode($indices));
echo("\n");

if (gettype($indices) == "array"){
	print(json_encode($numberList[$indices[0]]));
	echo("\n");
	print(json_encode($numberList[$indices[1]]));
	echo("\n");
}

?>
